myapp/spec/models/tasks_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+describe Task, type: :model do
+  describe 'validations' do
+    subject { build(:task) }
+
+    it { should validate_presence_of(:title) }
+    it { should validate_presence_of(:description) }
```

Comment:
only added model specs here as I have wrote the system specs for error messages in previous PR

https://github.com/Fablic/training/blob/d1415fdd35194ec03bc9347368dbc1c06c649b0c/myapp/spec/system/tasks_spec.rb#L91-L104